src/reporter.ts

```diff
@@ -35,7 +36,7 @@ export class Reporter {
   private testItems: TestItem[];
 
   constructor(config: ReportPortalConfig) {
-    this.noColors = true;
+    this.noColors = false;
```

Comment:
Why did you change this value?

Reply:
It's just a marker that controls the console output, colored or not.
So I decided that colored output will be more useful :)